src/sentry/tasks/sentry_apps.py

```diff
@@ -51,6 +51,10 @@
 
 TYPES = {"Group": Group, "Error": Event}
 
+DEBUG_INTERNAL_SENTRY_APPS = [
+    "a1f6cc7b-1201-4f4b-bf9e-4f6040385ab8",
+]
```

Comment:
If there is better way to do this, please let me know.

Reply:
So usually I'd say that you should make `DEBUG_INTERNAL_SENTRY_APPS` read from configs so that anyone running Sentry themselves can use this debugging if they want to. But what would happen if we just started logging this _always_?

Reply:
This would be a pretty large new log, since it triggers for the `error.created` event, on every internal sentry-app-installation. I limited it to just the installations as part of that list so that it wouldn't overwhelm logging just for debug purposes.

That said, config is a good idea, and makes more sense -- I'll look into that